packages/block-editor/src/hooks/border.js

```diff
@@ -2,7 +2,7 @@
  * WordPress dependencies
  */
 import { getBlockSupport } from '@wordpress/blocks';
-import { PanelBody } from '@wordpress/components';
+import { PanelBody, ALL_CSS_UNITS } from '@wordpress/components';
```

Comment:
That's a weird thing to expose in components. Why do we need this? and also why we never needed such thing for other things than "border"?

Reply:
`border` is the only thing that uses `parseUnit`, and it doesn't even reuse the function we have in the `unit-control`, it adds its own function. I'd rather simply delete the duplicate `parseUnit` function here and reuse the one from components, it simply felt out of scope for this PR. Should we do that?

Reply:
Let's try it 👍

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/32482/commits/afdf29d0318e16ac553e6951801e0a8efaaf02a7 :+1:

Reply:
It seems something that `UnitControl` requires from consumers, I brought this up at https://github.com/WordPress/gutenberg/pull/31483#discussion_r645685944